### ISSUE TITLE
Using Arduino Uno, CNC shield and TMC2208 in UART

### ISSUE THREAD

**Issue by AndyBerllan** [teemuatlut/TMCStepper#261]
Is it necessary to add 1K Ohm on a Y cable for the Tx line or does the CNC shield already have this?

**Comment by jasonmnemonic** [comment]
Yes to 1kOhm. You will need to look at their datasheet on which line. I cannot remember if it is at Tx or Rx side.
